common/io/Serial.cpp

```diff
@@ -247,6 +286,16 @@ void ReleaseUUCPLock(const std::string &path) {
       OLA_INFO << "Released " << lock_file;
     }
   }
+#endif  /* else no action needed */
```

Comment:
Please add an else and at least an OLA_DEBUG if not OLA_WARN, so the logs will show the function incorrectly getting called.

Reply:
No longer relevant, I think.  ReleaseUUCPLock now unconditionally tries a UUCP-style unlock, so the error should be obvious if it gets called instead of the new ReleaseSerialPortLock.

Reply:
Thanks, although please can we move the equivalent debug call to the new ReleaseSerialPortLock function instead please.

Reply:
I've added an OLA_INFO message for the flock() path in ReleaseSerialPortLock.  Is that what you meant?  Or, did you mean to add a return value to ReleaseUUCPLock and check it in ReleaseSerialPortLock?

Reply:
No that was what I meant, moving where the log message is generated.